### PR TITLE
Feat: homepage added

### PR DESCRIPTION
**Pull request by Sen-442b**
**Homepage** (Unresponsive)

### CODE REVIEW

**Comment by Sen-442b** [comment]
image
image

File: screens/homepage/source/style.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  
+}
+
+
+.primary-bg{
+  background-color: var(--background-theme);
+}
+.padding-lrg{
+  padding: 1.5em;
+}
```
Comment (BelideNikhil) [inline screens/homepage/source/style.css]: save this at the top under a comment saying utils

**Review by BelideNikhil — APPROVED** [review]
Remove the empty space and the commented stuff as well if not needed in style.css

**Review by kiranpratyush — APPROVED** [review]

File: screens/homepage/source/style.css (hunk 1 above)
Comment (Sen-442b) [reply screens/homepage/source/style.css]: Yeah it was supposed to be a part of  component library, thanks for pointing out

**Pull request merged**